Market/plugin.py

```diff
@@ -1185,6 +1185,63 @@ def ticker(self, irc, msg, args, optlist):
     ticker = wrap(ticker, [getopts({'bid': '','ask': '','last': '','high': '',
             'low': '', 'avg': '', 'vol': '', 'currency': 'currencyCode', 'market': 'something'})])
 
+#    Keep this for nostalgia reasons, possibly repurpose later
```

Comment:
After discussion with nanotube, this may be recoded to say something else, or be repurposed.  This was selectively readded to the Market plugin after stripping out other Gox functions.